src/EA.Weee.Web/Areas/AatfReturn/Mappings/ToViewModel/ReturnToObligatedViewModelMap.cs

```diff
@@ -76,10 +76,10 @@ public ObligatedViewModel Map(ReturnToObligatedViewModelMapTransfer source)
 
             if (source.PastedData != null)
             {
-                var obligatedPastedValues = new ObligatedPastedValues();
```

Comment:
Not your change, but Line 61's SiteName Null Check doesn't have a test

Reply:
Added unit test